OpenRA.Mods.Common/UpdateRules/UpdatePath.cs

```diff
@@ -64,7 +64,6 @@ public class UpdatePath
 
 			new UpdatePath("release-20180307", new UpdateRule[]
```

Comment:
This needs to be linked to the next path by inserting `"playtest-20180729",` before the `new UpdateRule[]`.